src/sparseml/pytorch/sparsification/quantization/quantize.py

```diff
@@ -232,16 +305,26 @@ def convert_module_qat_from_schemes(module: Module):
     configure_module_default_qconfigs(module)
 
     # set modules with proper qconfigs to QAT mode
-    mapping = _get_qat_module_mappings()
+    convert_kwargs = (
+        dict(convert_custom_config_dict={})  # do not let torch override any qconfigs
+        if version.parse(torch.__version__) >= version.parse("1.8.0")
+        else {}
+    )
     torch_quantization.convert(
-        module, mapping=mapping, inplace=True, remove_qconfig=False
+        module,
+        mapping=_get_qat_module_mappings(),
+        inplace=True,
+        remove_qconfig=False,
+        **convert_kwargs,
     )
-    torch_quantization.add_observer_(module, non_leaf_module_list=set(mapping.values()))
```

Comment:
So the new add_output_activation_observers replaces this call?

Reply:
yup